### DISCUSSION TITLE
How to include selected components for the project?

### DISCUSSION THREAD

**Discussion by gplhegde** [saadeghi/daisyui#1573] [Q&A]
I want to include only selected components for my project. Currently, all the components are getting included and tailwind pruning does not seem to work on daisyui components. Is there a way in which I can include only selected components, say card, button?

**Comment by saadeghi** [comment]
If you're using Tailwind CSS only generates styles for the class names it finds in your markup files [specified in `content` object in `tailwind.config.js`](https://tailwindcss.com/docs/content-configuration).  
If you did this setup correctly, there will be (_almost_) no unused styles in your CSS.  
I say "_almost_" because Tailwind CSS generates style for any strings it finds. For example if you don't have `<div class"alert">` but you have the word `alert` inside your files, `.alert` class will be generated.  

If your config is correct but you still have unwanted styles, I need to see your setup. please provide a reproduction repo so I can see it.
